### PR TITLE
Pj/fix/username login

### PR DESCRIPTION
**Pull request by pablisch**

### CODE REVIEW

File: api/models/token_generator.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ class TokenGenerator {
       iat: Math.floor(Date.now() / 1000),
       
       // Set the JWT token to expire in 10 minutes
-      exp: Math.floor(Date.now() / 1000) + (10 * 60)
+      // exp: Math.floor(Date.now() / 1000) + (10 * 60) // REAL LINE
+      exp: Math.floor(Date.now() / 1000) + (100 * 60) // EXTENDED TOKEN LINE
```
Comment (tomcarmichael) [inline api/models/token_generator.js:12]: oh wow... bravo, we salute you Pablo

File: frontend/src/components/navbar/NavBar.js [hunk 2]
```diff
@@ -25,21 +25,21 @@
...
           <>
-           <nav className='nav-bar'>
-           <h2>Farcebook</h2>
-             <div className="username-format">{username}</div>
-             <div className='nav-item' onClick={logout}><a href='/logout'>Logout</a></div>
-           </nav>
+          <nav className='nav-bar'>
+          <h2>Farcebook</h2>
+            <div className="username-format">{username}</div>
+            <div className='nav-item' onClick={logout}><a href='/login'>Logout</a></div>
```
Comment (tomcarmichael) [inline frontend/src/components/navbar/NavBar.js:31]: Is this a typo as link href is to /login but label says Logout?

File: api/controllers/tokens.js [hunk 3]
```diff
@@ -4,10 +4,11 @@ const TokenGenerator = require("../models/token_generator")
 const SessionsController = {
 
   Create: (req, res) => {
-    const email = req.body.email;
+    // const email = req.body.email;
+    const username = req.body.username;
     const password = req.body.password;
 
-    User.findOne({ email: email }).then(async (user) => {
+    User.findOne({ username: username }).then(async (user) => {
```
Comment (tomcarmichael) [inline api/controllers/tokens.js:11]: Great work - venturing into the darkest corners of the code base ..!

**Review by tomcarmichael — CHANGES_REQUESTED** [review]
This is gonna break so many tests ... heh

Fantastic work though. I learned a lot by reviewing this.

Just a question re: a potential typo in the navbar

File: api/controllers/tokens.js (hunk 3 above)
Comment (pablisch) [reply api/controllers/tokens.js:11]: 😁 just following the breadcrumbs.

File: frontend/src/components/navbar/NavBar.js (hunk 2 above)
Comment (pablisch) [reply frontend/src/components/navbar/NavBar.js:31]: No, this is intentional. It's a logout button, hence logout, but currently takes the user to ./logout which is a dead route when really they want to go back to ./login

**Review by tomcarmichael — APPROVED** [review]
This made a lot more sense to my brain this morning than last night - apologies for missing the obvious.

Pulled branch and had a look too, good work.

Not sure why these cypress files are appearing now? But also dont think it matters too much

**Pull request merged**